ipatests/test_integration/test_cert.py

```diff
@@ -7,15 +7,54 @@
 related scenarios.
 """
 import ipaddress
+import pytest
 import re
 
+from ipaplatform.paths import paths
 from cryptography import x509
 from cryptography.x509.oid import ExtensionOID
 from cryptography.hazmat.backends import default_backend
 
 from ipatests.pytest_ipa.integration import tasks
 from ipatests.test_integration.base import IntegrationTest
 
+DEFAULT_RA_AGENT_SUBMITTED_VAL = '19700101000000'
+
+
+def get_certmonger_fs_id(input_str):
+    """Get certmonger FS ID
+    from the `getcert list -f /var/lib/ipa/ra-agent.pem` output
+    command output
+
+    :
+    :return request ID string
+    """
+    request_id = re.findall(r'\d+', input_str)
+    print(request_id)
+    return request_id[1]
+
+
+def get_certmonger_request_value(host, requestid, directive):
+    """Get certmonger submitted value from
+    /var/lib/certmonger/requests/<timestamp>
+
+    :return submitted timestamp value
+    """
+    result = host.run_command(
+        'grep id=\'%s\' /var/lib/certmonger/requests/*' % requestid
+    )
+    print(result.stdout_text)
+    print(result.stderr_text)
+    assert result.stdout_text is not None
+    filename = result.stdout_text.split(':')[0]
```

Comment:
This can be removed as grep with -l option will give filename only
#grep -l id=20200324074843 /var/lib/certmonger/requests/*
/var/lib/certmonger/requests/20200324074844

Reply:
fixed it